src/app/pages/Map/Map.tsx

```diff
@@ -56,19 +56,14 @@ export default function MapBox(): JSX.Element {
       zoom: 4.5,
     })
 
-    if (map.current) {
-      map.current.on('load', function () {
-        map.current?.resize()
-      })
-    }
-
     //add MapControl
     const mapControl = new mapboxgl.NavigationControl()
     map.current.addControl(mapControl)
 
     //initiallize Geocoder and set to Div
     const geocoder = new MapboxGeocoder({
       accessToken: mapboxgl.accessToken,
+      placeholder: '         find location',
```

Comment:
instead of using all the blanks here, could you maybe give the placeholder a padding-left instead?

Reply:
thats true, i need to configure this input anyway, in a next commit :)